src/Service/RequestProcessor.cpp

```diff
@@ -58,8 +61,11 @@ void RequestProcessor::run()
             }
             request_thread->wait();
 
-            /// 3. process committed request, single thread
+            /// 2. process committed request, single thread
             processCommittedRequest(committed_request_size);
+
+            /// 3. process error requests
+            processErrorRequest();
```

Comment:
Need to get the number of current errors in advance, before moving to the pending queue, so as to ensure that the processed errors have been added to the pending queue.

Reply:
Now errors is not a queue, we can't handle n items in order. Maybe we should execute errors in order.